src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -294,19 +292,16 @@ function deleteQuestion(number) {
  * Allows users to cancel editing questions
  */
 function cancelEdit(number) {
-    if (number === -1) {
-        if (confirm('Are you sure you want to cancel adding this question?')) {
-            location.reload();
-        }
-    } else {
-        if (confirm('Are you sure you want to cancel your changes?')) {
-            location.reload();
-        }
+    var confirmationMsg = number === -1 ?
+            'Are you sure you want to cancel adding this question?' :
+            'Are you sure you want to cancel your changes?';
```

Comment:
The advice for parentheses in the style guide for JS is "Use sparingly and in general only where required by the syntax and semantics" and it's not syntactically required in this case.

As for ternary operator, it advises putting the operator at the end of the preceding line.
(This also applies to multi-line strings concatenated by +'s.)